src/lib/Gen/xml/XMLWriter.cpp

```diff
@@ -381,6 +385,74 @@ writeGuide(
     closeTemplate(I.Template);
 }
 
+void
+XMLWriter::
+writeAlias(
+    AliasInfo const& I)
+{
+    tags_.open(aliasTagName, {
+        { "name", I.Name },
+        { I.Access },
+        { I.id }
+        });
+
+    writeSourceInfo(I);
+
+    writeJavadoc(I.javadoc);
+
+    tags_.write("aliased", {}, {
+        {"name", toString(*I.AliasedSymbol)},
+        { I.AliasedSymbol->id }
+    });
+    tags_.close(aliasTagName);
+}
+
+void
+XMLWriter::
+    writeUsing(UsingInfo const& I)
+{
+    dom::String classStr;
+    switch (I.Class)
+    {
+    case UsingClass::Normal:
+        classStr = "using";
+        break;
+    case UsingClass::Typename:
+        classStr = "using typename";
+        break;
+    case UsingClass::Enum:
+        classStr = "using enum";
+        break;
+    case UsingClass::Namespace:
+        classStr = "using namespace";
+        break;
+    default:
+        MRDOCS_UNREACHABLE();
+    }
+
+    std::string qualifierStr;
+    if (I.Qualifier)
+    {
+        toString(*I.Qualifier);
```

Comment:
`toString(*I.Qualifier);` is a discarded-value expression... did you mean `qualifierStr = toString(*I.Qualifier);`?